api/pom.xml

```diff
@@ -35,6 +40,18 @@
             <artifactId>springfox-swagger-ui</artifactId>
             <version>${springfox.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-test</artifactId>
+            <version>${springboot.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
Unless this dependency is used in this PR, let's take it out and add back in once we actually need it.

Reply:
I thought this was needed but I guess it's not, I can remove